### ISSUE TITLE
Testing: $component->method() gives BadMethodCallException

### ISSUE THREAD

**Issue by ohnotnow** [livewire/livewire#573]
**Describe the bug**
I was following along the [test docs](https://laravel-livewire.com/docs/testing/) to try and test my component - but when I try and call a public method on the component that's being `Livewire::test()`'ed I get a `BadMethodCallException`.  Not sure why.

**To Reproduce**
The component :
```php
<?php

namespace App\Http\Livewire;

use App\DownloadedFile;
use Livewire\Component;

class FileList extends Component
{
    public $files;

    public function removeFile($fileId)
    {
        $file = DownloadedFile::findOrFail($fileId);
        $file->removeFromDisk();
        $file->delete();
    }

    public function render()
    {
        $this->files = DownloadedFile::latest()->get();
        return view('livewire.file-list');
    }
}
```

The initial scrappy test :
```php
    /** @test */
    public function we_can_remove_a_downloaded_file()
    {
        File::put('/tmp/test.txt', '');
        $file = factory(DownloadedFile::class)->create(['filename' => '/tmp/test.txt']);
        $fileList = Livewire::test(FileList::class);

        $fileList->removeFile($file->id); // this gives the BadMethodCallException

        $this->assertFalse(file_exists('/tmp/test.txt'));
        $this->assertDatabaseMissing('downloaded_files', ['id' => $file->id]);
    }
```

**Expected behavior**
Not to get a BadMethodCallException and for it to run the function.

**Desktop (please complete the following information):**
phpunit test  (the code does run fine 'for real' in the browser)

**Additional information**
I noticed the BadMethodCallException is saying `BadMethodCallException: Method Illuminate\Http\Response::removeFile does not exist` - if I `dd(get_class($fileList));` it comes back with `Livewire\Testing\TestableLivewire`, so I assume that's expected magic, but thought I'd mention it.

**Comment by calebporzio** [comment]
Thanks for submitting this issue, I think this was an oversight on my part.

The ability to run methods directly on test components was removed a few versions ago.

It must have gotten overlooked and not added to the changelog or upgrade guides. It's also still in the documentation.

I just removed it from the documentation, so the new testing page should demonstrate what's available: https://laravel-livewire.com/docs/testing

Sorry for the inconvenience and oversight.

I'll add a note to the next release about its removal.

Thanks! Closing for now.
